### ISSUE TITLE
Tutorial 1 Step 5 - test_folder created in wrong directory

### ISSUE THREAD

**Issue by Edward-Knight** [amblina/katacoda-scenarios#8]
Step 4 asks us to create the directory `/home/scrapbook/tutorial/step3/test_folder`,
but step 5 then asks us to work on `test_folder`, but says it's located at `/home/scrapbook/tutorial/test_folder`

**Comment by noellethenerd** [comment]
good point; btw I should mention step 5 is still in development and isn't live yet

**Comment by Edward-Knight** [comment]
I believe it is currently live?

**Comment by Edward-Knight** [comment]
Further, the start of step 5 assumes the user is in `/home/scrapbook/tutorial/step3` whereas they would be in `/home/scrapbook/tutorial/step3/test_folder` if they followed step 4 correctly

**Comment by Edward-Knight** [comment]
We had at least one learner struggle with this in today's session - they assumed they'd made a mistake in an earlier step and were now stuck.
